### PR TITLE
Converted the project to a package

### PR DESCRIPTION
**Pull request by AthulJoshi**
Could you please check on this?

### CODE REVIEW

**Comment by twentyse7en** [comment]
Oops, I didn't saw this. Please mention like this `@name`, so the user will get a mail. Or you can request a review .
image

**Comment by twentyse7en** [comment]
Thanks for the PR. Looks good!!

**Comment by twentyse7en** [comment]
Can you please create a `.gitignore` and add `__pycache__`. That will remove the `.pyc` files. [example](https://github.com/LiberTEM/LiberTEM/blob/16011aee5fab65fc2a4094ddb4e1a203f5d51b3e/.gitignore#L5)

**Comment by AthulJoshi** [comment]
> Can you please create a `.gitignore` and add `__pycache__`. That will remove the `.pyc` files. [example](https://github.com/LiberTEM/LiberTEM/blob/16011aee5fab65fc2a4094ddb4e1a203f5d51b3e/.gitignore#L5)

Sure, I'll do that

**Comment by AthulJoshi** [comment]
> Oops, I didn't saw this. Please mention like this `@name`, so the user will get a mail. Or you can request a review .
> image

The reviewer tab is empty..

**Comment by AthulJoshi** [comment]
I have added the gitignore @twentyse7en

**Comment by anandbaburajan** [comment]
> The reviewer tab is empty..

@AthulJoshi You've to click on the gear/settings icon

**Comment by AthulJoshi** [comment]
> > The reviewer tab is empty..
> 
> @AthulJoshi You've to click on the gear/settings icon

Idk why but there isnt a gear icon. Is it a problem of this PR?

**Comment by twentyse7en** [comment]
I think you have already tracked the `__pycache__` folder. `.gitignore` will not remove the tracked one. So for this time, you have to manually remove the `__pycache__` folder or try `git rm --cached __pycache__  -r`.

**Comment by anandbaburajan** [comment]
> > @AthulJoshi You've to click on the gear/settings icon
> 
> Idk why but there isnt a gear icon. Is it a problem of this PR?

My bad! You didn't have `write` access to this repository. If the icon is visible now, please test it out for us by asking @twentyse7en for a review. Thanks! :D

**Comment by AthulJoshi** [comment]
The pycache files are not being ignored. Do I have to add each of the files in the .gitignore?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+__pycache__/*
```
Comment (twentyse7en) [inline .gitignore]: I think `__pycache__/` is enough.

**Comment by twentyse7en** [comment]
> The pycache files are not being ignored. Do i have to add each of the files in the .gitignore?

No, It's there because you have already tracked those files earlier by `git add`, you need to manually delete those folders for now. Next time onwards they will be ignored.

**Comment by AthulJoshi** [comment]
Sorry for these many pushes. I should've checked everything first 😅.

**Comment by twentyse7en** [comment]
> Sorry for these many pushes. I should've checked everything first .

No worries : ) Multiple pushes are common when we work.

**Comment by twentyse7en** [comment]
Thanks for taking care of this.

**Pull request merged**